### PR TITLE
#217 - contact page

### LINKED ISSUE #217: Contact page
Należy zaimplementować zakładkę Kontakt
Link: https://xd.adobe.com/view/26f66876-cddd-469c-8b1f-b991dec40e55-73a5/screen/6818c986-601f-4459-baf2-91ff7b72b70b
Należy pamiętać o obsłudze wysyłanych wiadomości z tego miejsca

obraz

### PR DESCRIPTION
**Pull request by brzozka**
This should close #217 

Desktop preview:
Screenshot from 2024-06-14 11-30-53
Screenshot from 2024-06-14 11-33-46

Mobile preview:
Screenshot from 2024-06-14 11-31-15
Screenshot from 2024-06-14 11-31-22

Links to graphic design:
Desktop: [https://xd.adobe.com/view/26f66876-cddd-469c-8b1f-b991dec40e55-73a5/screen/6818c986-601f-4459-baf2-91ff7b72b70b](url)
Mobile: [https://xd.adobe.com/view/48e3dd96-96c0-4801-aee3-93a97687467e-1db0/screen/dd0d5103-5ac8-4fe4-8923-28ce2e09b490](url)

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
-      'sm': '640px',
-      'md': '768px',
-      'tablet': '850px',
-      'lg': '1024px',
-      'xl': '1280px',
+      sm: '640px',
+      md: '768px',
+      tablet: '850px',
+      lg: '1024px',
+      xl: '1280px',
```
Comment (kamilpiech97) [inline tailwind.config.js:16]: We must be consistent `2xl` or `'2xl'`

**Review by kamilpiech97 — CHANGES_REQUESTED** [review]

**Review by AleksandraKozubal — COMMENTED** [review]
1. On mobile location icon gets smaller:

image

2. In social media section please add youtube icon and change fb icon to `fa-square-facebook`

File: tailwind.config.js (hunk 1 above)
Comment (brzozka) [reply tailwind.config.js]: It was due to automatic linting, fixed.

File: lang/en/contact.php
Comment (AleksandraKozubal) [inline lang/en/contact.php:1]: So far, the approach adopted is that pages are translated in content.php with appropriate "prefix", like` "contact" => []...,`

File: lang/en/contact.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    "title" => "Write",
+    "title2" => "to Us!",
+    "company" => "Blumilk Sp. z o.o.",
+    "subtitle" => "Are you interested enough to want to discuss your project with us, exchange ideas, become our business partner, or work with us? We would love to talk with you!",
+    "form.email" => "E-mail",
```
Comment (AleksandraKozubal) [inline lang/en/contact.php:10]:
```suggestion
    "form" => [
        "email" => "E-mail",
    ],
```

Would be good to change it here and in other places also

File: lang/en/contact.php
Comment (brzozka) [reply lang/en/contact.php:1]: Fixed.

File: resources/views/contact.blade.php [hunk 3]
```diff
@@ -1,5 +1,213 @@
...
+                                class="block text-xs font-semibold leading-6 text-gray-700">{{ __('content.contact.form.message') }}</label>
+                            <div class="mt-2.5">
+                                <textarea id="message" name="message" rows="4"
+                                    class="block w-full xl:w-[90%] rounded-md border-0 px-3.5 py-2 min-h-12 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"></textarea>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="flex items-center mt-2 xl:w-[90%]">
+                        <input type="checkbox" name="policyBox" id="policyBox" class="w-3 h-3">
+                        <p class="text-xs font-normal text-gray-600 ml-2 pt-4">Akceptuję regulamin oraz <a href=""
```
Comment (AleksandraKozubal) [inline resources/views/contact.blade.php:70]: Would be nice to translate this whole line.

File: resources/views/contact.blade.php [hunk 4]
```diff
@@ -1,5 +1,213 @@
...
+        <a href="https://linkedin.com/company/blumilksoftware" class="text-gray-700" target="_blank">
+            <span class="sr-only">LinkedIn</span>
+            <i class="fa-brands fa-linkedin text-xl lg:text-2xl"></i>
+        </a>
+        <a href="https://www.youtube.com/@blumilksoftware" class="text-gray-700" target="_blank">
+            <span class="sr-only">Youtube</span>
+            <i class="fa-brands fa-square-youtube text-xl lg:text-2xl"></i>
+        </a>
+        <a href="https://www.facebook.com/blumilksoftware/" class="text-gray-700" target="_blank">
+            <span class="sr-only">X</span>
```
Comment (AleksandraKozubal) [inline resources/views/contact.blade.php:170]:
```suggestion
            <span class="sr-only">Facebook</span>
```

File: resources/views/contact.blade.php [hunk 5]
```diff
@@ -1,5 +1,213 @@
...
+            <i class="fa-brands fa-facebook text-xl lg:text-2xl"></i>
```
Comment (AleksandraKozubal) [inline resources/views/contact.blade.php:171]:
```suggestion
            <i class="fa-brands fa-square-facebook text-xl lg:text-2xl"></i>
```

File: resources/views/contact.blade.php [hunk 6]
```diff
@@ -1,5 +1,213 @@
...
+                    <a href="https://linkedin.com/company/blumilksoftware" class="text-gray-700" target="_blank">
+                        <span class="sr-only">LinkedIn</span>
+                        <i class="fa-brands fa-linkedin text-xl lg:text-2xl"></i>
+                    </a>
+                    <a href="https://www.youtube.com/@blumilksoftware" class="text-gray-700" target="_blank">
+                        <span class="sr-only">Youtube</span>
+                        <i class="fa-brands fa-square-youtube text-xl lg:text-2xl"></i>
+                    </a>
+                    <a href="https://www.facebook.com/blumilksoftware/" class="text-gray-700" target="_blank">
+                        <span class="sr-only">X</span>
```
Comment (AleksandraKozubal) [inline resources/views/contact.blade.php:143]:
```suggestion
                        <span class="sr-only">Facebook</span>
```

**Review by AleksandraKozubal — CHANGES_REQUESTED** [review]

File: resources/views/contact.blade.php (hunk 6 above)
Comment (brzozka) [reply resources/views/contact.blade.php]: Fixed.

File: resources/views/contact.blade.php (hunk 3 above)
Comment (brzozka) [reply resources/views/contact.blade.php]: Moved the text to content.php, please check if it's split correctly.

**Review by AleksandraKozubal — APPROVED** [review]

File: lang/en/content.php [hunk 7]
```diff
@@ -174,4 +174,32 @@
...
+        "location" => [
+            "address" => "ul. Najświętszej Marii Panny 5f/5",
+            "PostalCode" => "59-220 Legnica",
+        ],
+        "phone" => "+48 780 142 367",
+        "email" => "office@blumilk.pl",
+        "info" => [
+            "NIP" => "NIP 6912551135",
+            "KRS" => "KRS 0000866755",
+            "REGON" => "REGON 38737620",
```
Comment (kamilpiech97) [inline lang/en/content.php:196]: I would like to keep only these numbers:
```suggestion
            "NIP" => "6912551135",
            "KRS" => "0000866755",
            "REGON" => "38737620",
```

**Review by kamilpiech97 — CHANGES_REQUESTED** [review]

File: lang/en/content.php (hunk 7 above)
Comment (brzozka) [reply lang/en/content.php:196]: So, should I delete the text and add it statically to the webpage, or split it into a number and description?

**Review by EwelinaSkrzypacz — COMMENTED** [review]
1. For me, when user clicks text next to checkbox, it should checked. Now I have to aim to this little checkbox and it's a bit uncomfortable. 
2. As I understand, now I can't send a message? In issue description was information about sending message implementation. So, maybe we should another issue for that, just to not forget about making this?

**Review by EwelinaSkrzypacz — APPROVED** [review]

File: resources/views/contact.blade.php [hunk 8]
```diff
@@ -1,5 +1,216 @@
...
+        </a>
+        <a href="https://www.facebook.com/blumilksoftware/" class="text-gray-700" target="_blank">
+            <span class="sr-only">Facebook</span>
+            <i class="fa-brands fa-square-facebook text-xl lg:text-2xl"></i>
+        </a>
+    </div>
+    <script>
+        document.addEventListener('DOMContentLoaded', function() {
+            mapboxgl.accessToken =
+                'pk.eyJ1Ijoia3Jld2FrIiwiYSI6ImNramhmNWI3NjQ2Mm0yeW5xaHUxbmFvMTAifQ.pML1FJHSbrdqOGUvlnd1xw'
```
Comment (AleksandraKozubal) [inline resources/views/contact.blade.php:180]: What do you think about putting thus into config file?

Comment (kamilpiech97) [reply resources/views/contact.blade.php:180]: Or `.env` file

Comment (brzozka) [reply resources/views/contact.blade.php]: During testing I totally forgot about the token. I moved it to .env file.

**Review by AleksandraKozubal — DISMISSED** [review]

File: lang/en/content.php (hunk 7 above)
Comment (kamilpiech97) [reply lang/en/content.php]: What do u want

**Review by kamilpiech97 — DISMISSED** [review]

File: .env.example [hunk 9]
```diff
@@ -5,6 +5,9 @@ APP_KEY=base64:sCsJw8z+d/4ymp0OvzSip2h4Vp2hZZhpV2uOxgTqP94=
 APP_DEBUG=true
 APP_URL=http://website.blumilk.localhost
 
+MAPBOX_ACCESS_TOKEN="pk.eyJ1Ijoia3Jld2FrIiwiYSI6ImNramhmNWI3NjQ2Mm0yeW5xaHUxbmFvMTAifQ.pML1FJHSbrdqOGUvlnd1xw"
```
Comment (EwelinaSkrzypacz) [inline .env.example:8]:
```suggestion
MAPBOX_ACCESS_TOKEN=
```

File: .env.example [hunk 10]
```diff
@@ -5,6 +5,9 @@ APP_KEY=base64:sCsJw8z+d/4ymp0OvzSip2h4Vp2hZZhpV2uOxgTqP94=
...
+MAPBOX_STYLE="mapbox://styles/krewak/ckjhf6g1qbww719p1e33pgm84"
```
Comment (EwelinaSkrzypacz) [inline .env.example:9]:
```suggestion
MAPBOX_STYLE=
```

**Review by EwelinaSkrzypacz — CHANGES_REQUESTED** [review]

File: .env.example (hunk 9 above)
Comment (brzozka) [reply .env.example]: Fixed.

File: .env.example (hunk 10 above)
Comment (brzozka) [reply .env.example]: Fixed.

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by kamilpiech97 — APPROVED** [review]

**Pull request merged**